pfb-analysis/import/import_neighborhood.sh

```diff
@@ -45,9 +44,10 @@ NB_POSTGRESQL_PASSWORD - Default: gis
 function import_and_transform_shapefile() {
     IMPORT_FILE="${1}"
     IMPORT_TABLENAME="${2}"
+    IMPORT_SRID="${3:-4326}"
```

Comment:
I'm surprised these `IMPORT_SRID` changes are in here. Wasn't that change made awhile ago?